client/src/components/History/Content/ContentItem.vue

```diff
@@ -52,6 +49,13 @@
                     @unhide="$emit('unhide')" />
             </div>
         </div>
+        <div v-if="isDataset" class="p-1 all-tags">
```

Comment:
@ahmedhamidawan This is why that test is failing; we do expose/display tags for Collections in the beta history (just like any other content item, you will want to see these at the top level).